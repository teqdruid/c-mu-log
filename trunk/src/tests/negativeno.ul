// tests to test whether negative no.s are accepted
wall(-4,5);
wall(?,-2);
wall(-1,-1);
size(-10,-10);

main(){ 
@print("wall(",$x,",",$y,")");
wall($x,$y);         	 
}
