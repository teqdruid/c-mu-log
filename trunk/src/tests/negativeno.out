wall(-4,5)

 ^^^  Solution  ^^^ 

wall(Any,-2)

 ^^^  Solution  ^^^ 

wall(-1,-1)

 ^^^  Solution  ^^^ 

No more solutions
